src/main/java/Jarvis.java

```diff
@@ -0,0 +1,143 @@
+                    String date = userInput.substring(divisor + 4);
+                    taskList.add(new Deadline(description, date));
+                    System.out.println("Got it. I've added this task:\n " + taskList.get(Task.count - 1)
+                            + "\nNow you have " + (Task.count) + " tasks in the list.");
+                    continue;
+                }
+                //if userinput equals event add new event task to list
+                if (userInput.length() > 7 && userInput.substring(0, 5).equals("event")) {
+                    int divisor = userInput.indexOf("/at");
+                    String description = userInput.substring(6, divisor - 1);
+                    String date = userInput.substring(divisor + 4);
+                    taskList.add(new Event(description, date));
+                    System.out.println("Got it. I've added this task:\n " + taskList.get(Task.count - 1)
+                            + "\nNow you have " + (Task.count) + " tasks in the list.");
+                    continue;
+                }
+                throw new JarvisException("I'm sorry, but I don't know what that means");
+            }
+            try {
+                File dir = new File("data");
+                File myFile = new File(dir, "taskList.txt");
+                myFile.createNewFile();
+                FileWriter myWriter = new FileWriter(myFile);
+                for (int i = 0; i < taskList.size(); i++ ) {
+                    Task curr = taskList.get(i);
+                    if (curr instanceof Deadline) {
+                        if (curr.isDone) {
+                            myWriter.write("D" + " | 1 | " + curr.description + " | " + ((Deadline) curr).by + "\n");
+                        } else {
+                            myWriter.write("D" + " | 0 | " + curr.description + " | " + ((Deadline) curr).by + "\n");
+                        }
+                    } else if (curr instanceof Event) {
+                        if (curr.isDone) {
+                            myWriter.write("E" + " | 1 | " + curr.description + " | " + ((Event) curr).at + "\n");
+                        } else {
+                            myWriter.write("E" + " | 0 | " + curr.description + " | " + ((Event) curr).at + "\n");
+                        }
+                    } else {
+                        if (curr.isDone) {
+                            myWriter.write("T" + " | 1 | " + curr.description + "\n");
+                        } else {
+                            myWriter.write("T" + " | 0 | " + curr.description + "\n");
+                        }
+                    }
+                }
+                myWriter.close();
+            } catch (IOException e) {
+                System.out.println("An error occurred.");
+                e.printStackTrace();
+            }
+            System.out.print(farewell);
+        }
+        catch (JarvisException e) {
+            System.out.println((e));
```

Comment:
Perhaps it would be better to abstract the running of the chatbot into a method for itself, which can instead be called here to ensure the bot continues running when facing issues.